### PR TITLE
[gui-test][full-ci] enable invalid-name check

### PR DESCRIPTION
**Pull request by saw-jan**
Enable `invalid-name` lint rule. This rule checks the naming convention

Here are the set config:
```env
attr-naming-style=snake_case
class-const-naming-style=UPPER_CASE
class-naming-style=PascalCase
const-naming-style=UPPER_CASE
function-naming-style=snake_case
method-naming-style=snake_case
module-naming-style=any
variable-naming-style=snake_case
```

### CODE REVIEW

File: test/gui/shared/scripts/pageObjects/PublicLinkDialog.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ class PublicLinkDialog:
     defaultExpiryDate = ""
```
Comment (ishabaral) [inline test/gui/shared/scripts/pageObjects/PublicLinkDialog.py:95]:
```suggestion
    DEFAULT_EXPIRY_DATE = ""
```

Comment (saw-jan) [reply test/gui/shared/scripts/pageObjects/PublicLinkDialog.py:95]: hmm, the linter didn;t catch the class members

Comment (saw-jan) [reply test/gui/shared/scripts/pageObjects/PublicLinkDialog.py:95]: this rule was missing `class-attribute-naming-style`

File: test/gui/.pylintrc [hunk 2]
```diff
@@ -39,10 +37,13 @@ unsafe-load-any-extension=no
 
 [BASIC]
 attr-naming-style=snake_case
+; TODO: enable it later
+; class-attribute-naming-style=snake_case
```
Comment (saw-jan) [inline test/gui/.pylintrc:41]: looks like it requires a bit of restructuring class variables. So, I will enable it later (next PR)

**Review by ishabaral — APPROVED** [review]

**Review by ishabaral — APPROVED** [review]

**Pull request merged**